Original patch from: gentoo/src/patchsets/glibc/2.9/1018_all_glibc-awk-in-C-locale.patch

-= BEGIN original header =-
http://bugs.gentoo.org/252802

2008-12-29  Mike Frysinger  <vapier@gentoo.org>

	* iconvdata/Makefile (iconv-rules): Use LC_ALL=C when running awk script.

-= END original header =-

---
 iconvdata/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/iconvdata/Makefile
+++ b/iconvdata/Makefile
@@ -286,7 +286,7 @@
 	{ echo $(filter-out lib%, $(modules)); \
 	  echo 8bit $(gen-8bit-modules); \
 	  echo 8bit-gap $(gen-8bit-gap-modules); } | \
-	$(AWK) 'NR == 1 { \
+	LC_ALL=C $(AWK) 'NR == 1 { \
 		  for (i = 1; i <= NF; i++) { \
 		    printf "%s-routines := %s\n", $$i, tolower($$i); \
 		    printf "%s-map := gconv.map\n", $$i; \
